src/_atomic_data.cpp

```diff
@@ -18,25 +17,6 @@ void pyne::_load_atomic_mass_map_memory() {
   } else { 
     _insert_abund_map();
   }
-
-  // calculate the atomic_masses of the elements
-  std::map<int,double> :: iterator it;
-
```

Comment:
Oh yeah, you are right. I didn't actually remove this. It was picked up from elsewhere. Not sure who removed it.